Lib/test/test_argparse.py

```diff
@@ -4809,6 +4809,20 @@ def test_issue_15906(self):
         args = parser.parse_args([])
         self.assertEqual(args.test, [])
 
+# =======================
+# unhashable type tests
+# =======================
+
+class TestUnhasableTypeError(TestCase):
+
+    def test_unhashable_type_noerror(self):
+        "_registry_get handles unhashable TypeError"
```

Comment:
As a convention, docstrings should be with """, as compare to the other test classes, docstrings are being places on the class level, it would be nice to have it like that too. :)